<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1637</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16776965</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dashboard {
        background-color : #F6F7F8;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame{
        background-color : white;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color : #1193d4;
border-radius : 30%;;
border-radius : 15px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="ressources.qrc">:/new/icons/icons/night_shelter_24dp_E3E3E3_FILL0_wght400_GRAD0_opsz24.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color : black;
font : 17.5px bold;</string>
              </property>
              <property name="text">
               <string>Home Manager</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="Menu">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton::hover {
	background-color : #E7F4FB;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btn_dashboard">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#btn_dashboard {
border : none;
text-align : left;
padding : 0 0 0 20px;
color : #1194d4;
font-weight : bold;
font-size : 15.5px;
background-color  : #E7F4FB;
border-radius : 10px;
}

#btn_dashboard::icon{
		border : 2px solid black;
}</string>
              </property>
              <property name="text">
               <string>Dashboard</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_reservation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#btn_reservation {
border : none;
text-align : left;
padding : 0 0 0 20px;
color : black;
font-size : 15.5px;
border-radius : 10px;
}</string>
              </property>
              <property name="text">
               <string>Reservations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_room">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#btn_room {
border : none;
text-align : left;
padding : 0 0 0 20px;
color : black;
font-size : 15.5px;
border-radius : 10px;
}</string>
              </property>
              <property name="text">
               <string>Rooms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_guest">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#btn_guest {
border : none;
text-align : left;
padding : 0 0 0 20px;
color : black;
font-size : 15.5px;
border-radius : 10px;
}</string>
              </property>
              <property name="text">
               <string>Guests</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_report">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#btn_report {
border : none;
text-align : left;
padding : 0 0 0 20px;
color : black;
font-size : 15.5px;
border-radius : 10px;
}</string>
              </property>
              <property name="text">
               <string>Reports</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16776065</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#scrollArea {
	border : none;
}</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1347</width>
           <height>1020</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color : #F6F7F8;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true">#page{
        background-color : transparent;
}</string>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QWidget" name="widget_title" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : black;</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="styleSheet">
                    <string notr="true">color : black;
font-weight : bold;
font-size : 30px;
</string>
                   </property>
                   <property name="text">
                    <string>Dashboard</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="styleSheet">
                    <string notr="true">color : rgb(107, 114, 128);
font-size : 15px;</string>
                   </property>
                   <property name="text">
                    <string>Overviews of today's operation and keys metrics</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_stats" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="styleSheet">
                    <string notr="true">font-size : 20px;
font-weight : bold;
color : black;</string>
                   </property>
                   <property name="text">
                    <string>Quick Stats</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame">
                   <property name="styleSheet">
                    <string notr="true">QWidget {
		border-radius : 20px;
}</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QWidget" name="widget_availability" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>350</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>351</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">#widget_availability {
        padding : 24px;
        color : black;
        background-color : white;
		padding : 20px;

}

QWidget {
        color : black;

}</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_9">
                       <item>
                        <widget class="QLabel" name="label_10">
                         <property name="styleSheet">
                          <string notr="true">font : 500 14px;
background-color: white;
color : black;
padding : 10px;
</string>
                         </property>
                         <property name="text">
                          <string>Available rooms</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_11">
                         <property name="styleSheet">
                          <string notr="true">font : 700 30px;
background-color : white;
padding : 5px;</string>
                         </property>
                         <property name="text">
                          <string>25</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Orientation::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_occupying" native="true">
                      <property name="minimumSize">
                       <size>
                        <width>350</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>351</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">#widget_occupying {
        padding : 24px;
        color : black;
        background-color : white;

}

QWidget {
        color : black

}</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_10">
                       <item>
                        <widget class="QLabel" name="label_18">
                         <property name="styleSheet">
                          <string notr="true">font : 500 14px;
background-color: white;
padding : 10px;</string>
                         </property>
                         <property name="text">
                          <string>Occupied rooms</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_19">
                         <property name="styleSheet">
                          <string notr="true">font : 700 30px;
background-color : white;
padding : 5px;</string>
                         </property>
                         <property name="text">
                          <string>75</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Orientation::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_checkins" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>350</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>350</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">#widget_checkins {
        padding : 24px;
        color : black;
        background-color : white;

}

QWidget {
        color : black

}</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_11">
                       <item>
                        <widget class="QLabel" name="label_22">
                         <property name="styleSheet">
                          <string notr="true">font : 500 14px;
background-color: white;
padding : 10px;</string>
                         </property>
                         <property name="text">
                          <string>Check-ins today</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_23">
                         <property name="styleSheet">
                          <string notr="true">font : 700 30px;
background-color : white;
padding : 5px;</string>
                         </property>
                         <property name="text">
                          <string>10</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>350</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="styleSheet">
                    <string notr="true">font-size : 20px;
font-weight : bold;
color : black;</string>
                   </property>
                   <property name="text">
                    <string>Today's Reservations</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableView" name="tableView_reservation">
                   <property name="styleSheet">
                    <string notr="true">QHeaderView::section {
    background-color: #F9FAFB;
    color: black;
    padding: 12px 24px;
text-align : left;
}

QTableView {
		background-color : white;
}

QTableView::item {
		border : none;
}</string>
                   </property>
                   <property name="alternatingRowColors">
                    <bool>false</bool>
                   </property>
                   <property name="showGrid">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>350</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="styleSheet">
                    <string notr="true">font-size : 20px;
font-weight : bold;
color : black;</string>
                   </property>
                   <property name="text">
                    <string>Rooms Occupancy</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableView" name="tableView_occupancy">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>350</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2"/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
